_2020_pages/venue.html

```diff
@@ -0,0 +1,134 @@
+        <p>The <a href="https://www.dus.com/en" class="ext" target="_blank">Airport Düsseldorf </a>is  the nearest
+          airport equipped with an  international transport network.  You can reach Münster from airport by  train in
+           about 90 minutes. For  more details see the Travel Service of
+           <a href="https://www.bahn.com/en/view/index.shtml" class="int">"Deutsche Bahn"</a>. Alternatively,
+           the Airport Münster-Osnabrück, which has  a significant smaller transport system, is located about 30 km
+           in the  north of Münster.</p>
+        <ul>
+          <li><strong>Airport Münster-Osnabrück --&gt; Münster (Westf.) Central Station</strong><br />
+          The <a href="https://www.fmo.de/en/" class="ext" target="_blank">Airport Münster-Osnabrück</a> is located
+          about 30 km north of Münster and offers daily scheduled flights to various destinations within Germany and
+          Europe. The airport offers a shuttle service to Münster's main railway station which runs approximately every
+           30 minutes. You can buy a ticket on the bus for about 5 Euro. The journey takes about 35 minutes.</li>
+          <li><strong>Dortmund Airport --&gt; Münster (Westf.) Central Station</strong><br />
+          There is a regional train that runs from Dortmund airport (Holzwickede station) to Münster main station. The
+          station in Dortmund-Holzwickede can be reached by a shuttle bus from the airport within 5 minutes. The complete
+          journey takes between 1 - 1.5 hours.</li>
+          <li><strong>Düsseldorf Airport --&gt; Münster (Westf.) Central Station</strong><br />
+          The nearest airport with a good international route network is Düsseldorf Airport.<br />
+          There is a direct train from Düsseldorf to Münster on platform 5. This takes just under 1.5 hours.</li>
+          <li><strong>Cologne/Bonn Airport --&gt; Münster (West) Central Station</strong><br />
+          When you arrive at Cologne/Bonn Airport, you have to change trains once. Either at Cologne Central Station,
+           Cologne Deutz Station or Essen Central Station. The whole journey takes about 2.5 hours.</li>
+          <li><strong>Frankfurt Airport --&gt; Münster (West) Central Station</strong><br />
+          To get from Frankfurt Airport to Münster, you can take a direct train from platform FERN 7. The journey takes
+          about 3.5 hours.<br />
+          Alternatively, you can also travel via Dortmund and change trains there. In this case you leave from platform
+          FERN 6 and need about 2.5 hours for the whole journey.</li>
+        </ul>
+
+      <h3>By public transport</h3>
+        <p>Münster is well connected by train and bus. See the website of
+            <a href="https://www.bahn.com/en/view/index.shtml" class="ext" target="_blank">"Deutsche Bahn" </a>
+            for more travel details.
+        </p>
+        <h4>From the train station to Seminarraumzentrum</h4>
```

Comment:
For some reason, `h4` is blue, but `h3` is black. Is this on purpose?

Reply:
Nope